.editorconfig

```diff
@@ -0,0 +1,18 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+max_line_length = 80
+
+[{Makefile,GNUmakefile.in}]
```

Comment:
Shouldn't this also include `GNUmakefile`?

Reply:
Fixed. I thought this is not that relevant as it is generated.

Reply:
The `GNUmakefile` in the root is generated, but the ones in e.g. `common/` and `decompressor/` are not.